app/main/views/direct_award.py

```diff
@@ -0,0 +1,193 @@
+from flask import jsonify, abort, current_app, request
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy import desc
+
+from dmapiclient.audit import AuditTypes
+from .. import main
+from ... import db
+from ...models import User, DirectAwardProject, AuditEvent, DirectAwardSearch
+from ...utils import (
+    get_json_from_request, get_int_or_400, json_has_required_keys, pagination_links,
+    get_valid_page_or_1, validate_and_return_updater_request, validate_user_can_access_direct_award_project_or_403)
+
+
+@main.route('/direct-award/projects', methods=['GET'])
+def list_projects():
+    user_id = get_int_or_400(request.args, 'user_id')
+    if user_id is None:
+        abort(400, "User ID not supplied")
```

Comment:
It might be something for our tech debt, but this bit of code is repeated everywhere, it would be nice to move it somewhere else and use it as a decorator like we do in other apps.